locales/en.yml

```diff
@@ -86,6 +89,10 @@ en:
           response: "%{name} (%{email}) is currently on call for %{schedule_name}"
           no_matching_schedule: "No matching schedules found for '%{schedule_name}'"
           no_one_on_call: "No one is currently on call for %{schedule_name}"
+        base_lookup:
+          response: "%{name} (%{email}) is on call in the base layer (%{layer_name}) of %{schedule_name} until %{schedule_end} (now=%{now})."
```

Comment:
since this is already a pretty long response, wondering if there is any reason to include the `now` bit at the end?

Reply:
Nah, I'll get rid of that. I put it in when I was debugging, and then it made sense to give the user a relative feel since timezones aren't particurly obvious. Eg

Pagerduty is set up in something like UTC-7. If you're in that timezone, that makes sense, and it's easy to assume that that is probably what you're looking at. If you're in another timezone it's less obvious.

Throw into the mix a few others timezones that get used for different layers to make it make sense for what they are representing, and there's room for confusion if it's not done right.

I'll remove this for now, and potentially revisit _a_ solution if this confusion eventuates after the next phase, which will give a more wholistic view.

Reply:
That's removed.